Tests/test_GenBank_unittest.py

```diff
@@ -432,6 +433,43 @@ def test_genbank_date_invalid(self):
             gb = SeqIO.read(handle, "gb")
             self.assertEqual(gb.annotations["date"], "01-JAN-1980")
 
+    def test_longer_locus_line(self):
+        """Check that we can read and write files with longer locus lines"""
+        # Create file
+        with open(path.join("GenBank", "DS830848.gb"), 'r') as inhandle:
+            data = inhandle.readlines()
+            data[0] = "LOCUS       AZZZAA02123456789 10000000000 bp    DNA     linear   PRI 15-OCT-2018\n"
+        with open(path.join("GenBank", "long_header_genbank_test.gb"), 'w') as outfile:
```

Comment:
Please don't use a named temp file under the ``Tests/`` folder, they tend not to get cleaned up properly and occasionally get checked into version control.

I would use a ``StringIO`` handle (via ``from Bio._py3k import StringIO``), or use ``tempfile`` if you really want an actual file.